client/src/components/game/phases/Events.vue

```diff
@@ -1,11 +1,25 @@
 <template>
   <div class="c-events container tour-event">
     <div class="wrapper row">
-      <div class="event-deck col-3 tour-event-deck">
-        <div class="topbar">
-          <p class="title">Event Deck</p>
+      <div
+        v-if="shouldShowAccomplishments"
```

Comment:
why are we deciding to interchange the active events section with active accomplishments?

Reply:
oh, i see what's happening now. so i'm unable to switch between active accomplishments and events until the last event. i think that functionality should be enabled on all events that are drawn

Reply:
@chrstngyn that can be easily enabled. nick had it set up originally to only show the active accomplishment cards during certain events, so i just kept that functionality. any thoughts @Nick42791 ?

Reply:
In the original implementation, I had the whole thing hidden entirely unless it was needed. It looks like the new one has you switch which view you can see, so it might be weird to have a button that you can't click _sometimes_.

I still think that having the accomplishments when they aren't needed isn't fantastic, so maybe we can get around it by making the accomplishments tab always clickable, but for events that they are very relevant in, the view switches automatically.

Reply:
on some events (from what i've noticed: personal gain, out of commission), it's not obvious what the event is unless there is an active events column on the left side. 

the orange header "active events: [event name here]" also doesn't update on personal gain and out of commission. not sure if this is true for other events. and it's not my first instinct to click on the active events popup to see what events are active.

i think it would be best to have both the tab switcher for active events + accomplishments, but set the active events tab as the default view. i understand the need for an active accomplishments tab, but i can also foresee confusion if i'm playing the game for the first time. we might be circling back to the problem where there's too much going on in the UI.

Reply:
for example, this is Personal Gain. the header should also display the name of the event, and i think it would be helpful to have that active events column to tie that info together in the same container.

![image](https://user-images.githubusercontent.com/8737685/79911526-5b39aa80-83d5-11ea-8820-0da936a5f532.png)

Reply:
@chrstngyn i can look at the event title error, not sure why that's not working but i remember that being an issue in the play test

also, ideally, players should reference their accomplishment cards by clicking on their profile in the hud. is there a solution where we remove the accomplishment cards entirely and direct players to their profiles, or is there a strong reason to keep the cards there @Nick42791?

Reply:
I don't think that the ideal case is people checking their hud. It's an extra click, you have to remember the information, and we have the space for to put both sets of information the phase anyway.

The ability to switch between the event deck and the active accomplishments should always be given. 

By default, the event deck should be visible. 

Events in which accomplishments are relevant, namely Influences Select and Influences Draw, should either:

- switch to the accomplishments view but leave the option for the event deck view

- stay in the event deck view but highlight the active accomplishment view in some way. 


As a side note: I **don't** think that just having three panes for some events is a bad thing.

This would allow us to get all of the information across very cleanly. I know that you're worried about consistency, but the active accomplishments are special cased anyway, so I feel an exception is not out of line. If you feel like the view switching too abruptly is a problem, we could add an animation to help with that.

Reply:
@Nick42791 i think that's a good argument to include an option to view active accomplishments. although, if we want to do it this way, we would have to refactor the tutorial to explain the active accomplishments tab in relation to a specific event that a player would want to look at their accomplishments.

so i think for this implementation to be complete:
1. make a tutorial step for it creating information relationship between the active accomplishments and simulating an event where it would be helpful for a player to view that tab
2. when an event comes up where they should be referencing their accomplishments to make a decision on an event that requires a player to vote, create some type of tool tip/indicator that it would be helpful for a player to switch to that active accomplishments tab.

Reply:
ok, i just committed some changes:

- both 'Event Deck' and 'Accomplishments' tabs are always available in the events container
- the default view is the 'Event Deck'. i included a watcher function that switches the view back to 'Event Deck' if the marsEventsProcessed state value changes
- the "Active Event: None" title error is now fixed, it was a fun javascript comparison issue

Reply:
looks good! thanks @lfaivre